pkg/vm/memory/segments.go

```diff
@@ -67,7 +67,9 @@ func (s *MemorySegmentManager) RelocateMemory(relocationTable *[]uint) (map[uint
 				if err != nil {
 					return nil, err
 				}
-				relocatedMemory[relocatedAddr] = value
+				// Todo: fix this, should be a felt
+				val, _ := value.ToU64()
+				relocatedMemory[relocatedAddr] = uint(val)
```

Comment:
👀

Reply:
due to all the merging i can just return a felt because it breaks a lot of things, to fix it i would do a separate pull request.

Reply:
or you mean the u64 conversion?

Reply:
I forgot the writing the memory to a file pr was merged, yes it is indeed worth it to leave it to another PR